woocommerce/woocommerce
# Add Tooltip component and remove EnrichedLabel

### All Submissions:






### Changes proposed in this Pull Request:

This PR extracts the Tooltip portion from the EnrichedLabel and creates a separate component for that.  We have multiple use cases for the tooltip that fall outside of labels, so I think it's worthwhile for it to be its own component.

This PR also adds some accessibility keyboard handlers to the tooltip button, though admittedly this does not follow the traditional tooltip pattern and behaves more like a modal (since we can click content inside of the tooltip).

cc @octaedro want to make sure you're on board with this change and I'm not breaking things by removing the `EnrichedLabel`.

<img width="329" alt="Screen Shot 2022-10-10 at 8 57 12 AM" src="https://user-images.githubusercontent.com/10561050/194908164-0493d53e-5972-4657-838b-1dd553129ae5.png">
<img width="385" alt="Screen Shot 2022-10-10 at 8 57 07 AM" src="https://user-images.githubusercontent.com/10561050/194908167-eb9b6217-8e47-4a39-95f5-a52b788e9d83.png">


Closes #34997 .

### How to test the changes in this Pull Request:

1. Run `pnpm --filter=@woocommerce/storybook storybook`
2. Go to Experimental->Tooltip
3. Check that the stories work as expected
4. Attempt to open and close the tooltip via keyboard (enter/escape on focus)
5. Navigate to the new product management experience (Products -> Add new (MVP))
6. Make sure the tooltip next to `Feature this product` continues to work as expected

### Other information:




### FOR PR REVIEWER ONLY:
